client/src/app/components/stardust/AssociatedOutputs.tsx

```diff
@@ -22,7 +22,7 @@ interface AssociatedOutputsTableProps {
     /**
      * Address details
      */
-    addressDetails: IBech32AddressDetails;
+    addressDetails?: IBech32AddressDetails;
```

Comment:
Why make this optional ? Now we have to also care for additional cases inside the component.